src/collections/news/2020-05-28-layer5-and-hasicorp-launch-service-mesh-partnership/index.mdx

```diff
@@ -1,14 +1,13 @@
 ---
-layout: post
 title: "Layer5 and HashiCorp Launch Service Mesh Partnership"
-date:   2020-05-28 00:00:00 +08000
+date:   2020-05-28 00:00:00 +0800
 author: Lee Calcote
-type: news 
-permalink: /company/news/layer5-hashicorp-launch-service-mesh-partnership
-image: ./2020-05-28-layer5-hashicorp-launch-service-mesh-partnership/layer5-hashicorp.png
+thumbnail: ./2020-05-28-layer5-hashicorp-launch-service-mesh-partnership/layer5-hashicorp.png  
```

Comment:
remove the `2020-05-28-layer5-hashicorp-launch-service-mesh-partnership/` from the thumbnail Since both the `index.mdx` and the images are in the same folder.